JobConfig/beam/beam_defs_g4s1.fcl

```diff
@@ -329,3 +331,5 @@ services.message.destinations.log.categories.ArtReport.timespan : 300
 // Initialze seeding of random engines: do not put these lines in base .fcl files for grid jobs.
 services.SeedService.baseSeed         :  8
 services.SeedService.maxUniqueEngines :  20
+
+services.GeometryService.simulatedDetector.tool_type : "Mu2e"
```

Comment:
I believe that if you are doing everything else correctly then this should not be necessary - (not 100% certain about that).

Reply:
I removed this line in this file and in one other file.  The other file runs without it.

Reply:
Here is the story on that line.  If the fcl for the services contains  @local::Services.Sim  or @local::Services.SimAndReco, then you do not need this line - that definition is already there.  If the fcl for the GeometryService is just:
GeometryService : {  inputFile : "filename.txt" }

Then you need that line.  The preferred long term solution is that the owner of the file should retro-fit it to define services in the recommended way and then remove this line.